tests/layers/test_hyper_analysis_transformation_2d.py

```diff
@@ -14,4 +14,4 @@ class TestHyperAnalysisTransformation:
 
     x = torch.rand((28, 28, 5, 5))
 
-    assert transformation(x).shape == Size([28, 28, 2, 1])
+    assert transformation(x).shape == Size([28, 28, 2, 2])
```

Comment:
Is there a test for synthesis?

Reply:
Yep. `tests/layers/test_synthesis_transformation_2d.py`. It didn't need any modifications.